src/dioptra/restapi/v1/users/schema.py

```diff
@@ -37,3 +44,102 @@ class UserRefSchema(Schema):
         metadata=dict(description="URL for accessing the full User resource."),
         relative=True,
     )
+
+
+class UserMutableFieldsSchema(Schema):
+    """The schema for the mutable data fields in a User resource."""
+
+    username = fields.String(
+        attribute="name", metadata=dict(description="Username of the User.")
+    )
+
+    email = fields.String(
+        attribute="email", metadata=dict(description="Email of the User.")
+    )
+
+
+class UserSchema(UserMutableFieldsSchema):
+    """The schema for a User that is not the logged-in User."""
+
+    password = fields.String(
+        attribute="password",
+        metadata=dict(description="Password for the User resource."),
+        load_only=True,
+    )
+
```

Comment:
Also needs a `confirmPassword` field for initial user registration, so its `load_only=True`.

https://github.com/usnistgov/dioptra/blob/60af0a6aac3af7e4e9a47e16afa0e6877af9f05f/src/dioptra/restapi/v0/user/schema.py#L49-L55